core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -140,19 +140,19 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
     background-color: transparent;
     border: none;
     box-sizing: border-box;
-    color: #009688;
+    color: #000000;
```

Comment:
```suggestion
    color: #000;
```
here and ditto below

Reply:
Done